src/universal/utils/mode.js

```diff
@@ -0,0 +1,20 @@
+const mode = (array, getField = (item) => item) => {
```

Comment:
There wasn't a utility function in some library that could give you mode?

Reply:
it probably exists, but searching for "mode" or "math mode" or "arithmetic mode" on npm yields a lot of false positives. plus, i'm not doing a mode of primitives, so it's faster to just write my own.